commit/report.go

```diff
@@ -133,41 +315,44 @@ func (p *Plugin) validateReport(
 	lggr logger.Logger,
 	seqNr uint64,
 	r ocr3types.ReportWithInfo[[]byte],
-) (bool, cciptypes.CommitPluginReport, error) {
```

Comment:
The error type can now be used to distinguish between an invalid report and a validation error. This way we can pass additional context about why an error has occurred.